mock-project-backend/src/main/java/com/itsol/train/mock/repo/impl/TeamRepositoryImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.itsol.train.mock.repo.impl;
+
+import com.itsol.train.mock.entity.TeamEntity;
+import com.itsol.train.mock.repo.TeamRepository;
+import com.itsol.train.mock.utils.HibernateUtil;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.query.Query;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class TeamRepositoryImpl implements TeamRepository {
+    public Logger logger = LoggerFactory.getLogger(TeamRepositoryImpl.class);
+
+    @Override
+    public TeamEntity findByDefault() {
+        Session session = HibernateUtil.getSessionFactory().openSession();
+        try {
+            session.beginTransaction();
+            Query query = session.createQuery("select s from TeamEntity s where s.id = 0");
```

Comment:
It should be:
`Query<TeamEntity> query = session.createQuery("select s from TeamEntity s where s.id = 0");`